src/components/app/details/appDetails/utils.tsx

```diff
@@ -384,127 +386,233 @@ export function addQueryParamToGrafanaURL(url: string, appId: string | number, e
     return url;
 }
 
-export const processDeploymentStatusDetailsData = (data?: DeploymentStatusDetailsType): DeploymentStatusDetailsBreakdownDataType=>{
+export const processDeploymentStatusDetailsData = (data?: DeploymentStatusDetailsType): DeploymentStatusDetailsBreakdownDataType => {
   const deploymentData = {
       deploymentStatus: 'inprogress',
       deploymentStatusText: 'In progress',
       deploymentTriggerTime: data?.deploymentStartedOn || '',
       deploymentEndTime: data?.deploymentFinishedOn || '',
       deploymentError: '',
       triggeredBy: data?.triggeredBy || '',
+      nonDeploymentError: '',
       deploymentStatusBreakdown: {
           DEPLOYMENT_INITIATED: {
               icon: 'success',
-              displayText: 'Deployment initiated',
-              displaySubText: ` by ${data?.triggeredBy || ''}`,
+              displayText: `Deployment initiated by ${data?.triggeredBy || ''}`,
```

Comment:
can we show `by` only if we have `data?.triggeredBy`?

Reply:
done